engine/src/dot/http_rpc.rs

```diff
@@ -72,22 +73,23 @@ impl RpcClientT for PolkadotHttpClient {
 		_sub: &'a str,
 		_params: Option<Box<RawValue>>,
 		_unsub: &'a str,
-	) -> RpcFuture<'a, RpcSubscription> {
+	) -> RawRpcFuture<'a, RawRpcSubscription> {
 		unimplemented!("HTTP Client does not support subscription");
 	}
 }
 
 #[derive(Clone)]
 pub struct DotHttpRpcClient {
 	online_client: OnlineClient<PolkadotConfig>,
+	rpc_methods: LegacyRpcMethods<PolkadotConfig>,
```

Comment:
Can you make an issue to move to the new rpcs?

Reply:
Created the issue: PRO-1094

Reply:
> afaik they're not fully stable yet - at least not within subxt, I assume subxt just called it "legacy" so they don't have to rename things again

Just for my benefit, what do you mean but not fully stable? Do you mean they said somewhere the interfaces may be changed? Or do you mean something else.